cypress/e2e/components/atoms/button.cy.js

```diff
@@ -63,17 +65,11 @@ describe('Button component', () => {
         .find('textarea')
         .should('exist');
     });
-
-    it('closes code on click', () => {
```

Comment:
For whatever reason, these steps were causing tests to fail, even though the functionality works fine locally.

Given that this is testing CL itself, rather than our code, it seemed like a waste of time and energy to debug this right now, so I pulled out every instance of it across our tests.